pkg/shell/superuser.jsx

```diff
@@ -268,10 +267,13 @@ const LockDialog = ({ proxy, host }) => {
                onClose={D.close}
                footer={footer}
                title={_("Switch to limited access")}>
-            <>
-                <p>{_("Limited access mode restricts administrative privileges. Some parts of the web console will have reduced functionality.")}</p>
-                <p>{_("Your browser will remember your access level across sessions.")}</p>
-            </>
+            <Stack hasGutter>
+                {error && <ModalError dialogError={error} />}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17221-20221201-102811-6d301c18-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)